tests/general.rs

```diff
@@ -371,7 +371,7 @@ fn test_transport_message_exceeds_max_len() {
     let mut noise = NoiseBuilder::new(params).remote_public_key(&[0u8; 32]).build_initiator().unwrap();
 
     let mut buffer_out = [0u8; 65535*2];
-    noise.write_message(&[0u8; 0], &mut buffer_out).unwrap();
+    let _res = noise.write_message(&[0u8; 0], &mut buffer_out);
```

Comment:
what's the reason for this change to ignore the result of `write_message`?

Reply:
Because some implementations of `dh` may fail, and some may not. We can't for now check if result is `Err` or `Ok`.

Reply:
I'm a bit confused, wouldn't we want the test to fail if the dh fails?

Reply:
Nope, since it's an incorrect message (filled with only zeros), it's okay for dh to fail (as far as I know).